### PR TITLE
Add prismarine-item features

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

**Comment by rom1504** [comment]
ci failed

File: data/pc/common/features.json [hunk 1]
```diff
@@ -359,5 +359,35 @@
...
+    "description": "item serialization in older versions uses blockId field to show nullability by setting blockId to -1",
+    "versions": ["1.8_major", "1.12_major"]
+  },
+  {
+    "name": "preFlattening",
+    "description": "in older versions, enchant ids were used to store enchants in nbt",
+    "versions": ["1.8_major", "1.12_major"]
+  },
+  {
+    "name": "theFlattening",
```
Comment (rom1504) [inline data/pc/common/features.json:384]: pick a feature name related to what this does (like enchant)

Comment (u9g) [reply data/pc/common/features.json:384]: the flattening effected a lot of things, too many to point to one: https://minecraft.fandom.com/wiki/Java_Edition_1.13/Flattening

Comment (rom1504) [reply data/pc/common/features.json:384]: yes, I still think we should have one feature for one thing
features shouldn't be used only to identify a version range

btw I don't think this specific feature is that related with theFlattening (unlike blockStateId feature for example)

**Pull request merged**